source/MenuItem.hpp

```diff
@@ -57,17 +57,17 @@ namespace GTA
 		virtual void Change(bool right);
 		virtual void SetOriginAndSize(System::Drawing::Point topLeftOrigin, System::Drawing::Size size);
 
-		void UpdateText();
+		virtual void UpdateText();
 
 		virtual property MenuBase ^Parent;
 		virtual property System::String ^Caption;
 		virtual property System::String ^Description;
+		property UIText ^mText;
```

Comment:
I see what you mean, the reason is the result we want is a virtual UpdateText as it does bespoke formatting per component, (such as adding the numbers and < > signs at the end of the NumericScrollers and the like). As it speaks directly to the mText property I just made it public for now